samples/star/src/androidMain/kotlin/com/slack/circuit/star/MainActivity.kt

```diff
@@ -67,20 +70,32 @@ class MainActivity @Inject constructor(private val circuit: Circuit) : AppCompat
         persistentListOf(HomeScreen, petDetailScreen)
       }
 
+    val localCircuit =
+      circuit
+        .newBuilder()
+        // todo DI this
+        .addAnimatedScreenTransforms(
```

Comment:
Do you wanna handle this in this PR?

Reply:
Planning on doing so separately, given the size of this PR